new-dti-website/components/ui/carousel/useCarouselControls.ts

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useRef, useState } from 'react';
+import Autoplay from 'embla-carousel-autoplay';
+import type { CarouselApi } from './carousel';
+
```

Comment:
i also agree with @andrew032011 that the implementation of the carousels on the Product and About pages are a bit choppy.

Here's what it looks like for me:

https://github.com/user-attachments/assets/619207cf-baf6-466a-97e1-00a26e480ac3

The animation is very slow, and the image doesn't completely fit inside the white border before it scrolls again